common/autoconf/platform.m4

```diff
@@ -256,11 +256,13 @@ AC_DEFUN([PLATFORM_SETUP_TARGET_CPU_BITS],
         OPENJDK_TARGET_CPU=x86
       elif test "x$OPENJDK_TARGET_CPU_ARCH" = "xsparc"; then
         OPENJDK_TARGET_CPU=sparc
+      elif test "x$OPENJDK_TARGET_CPU_ARCH" = "xppc"; then
+        OPENJDK_TARGET_CPU=ppc
```

Comment:
I think `closed/openssl.gmk` needs to be updated to handle `ppc`.

Reply:
There is no need of openssl for 32-bit support.

Reply:
`configure` should fail if openssl is enabled for unsupported 32-bit platforms rather than build a 64-bit library.